Makefile.defs

```diff
@@ -79,6 +94,17 @@ else
     QFLAGS:=
 endif
 
+ifdef DARWIN
+GCCVERSION:=$(shell gcc -dumpversion | cut -c 1-3)
+ifeq ($(shell echo "$(GCCVERSION) < 4.8" | bc), 1)
```

Comment:
What's wrong with clang?

(For those who've never touched a Mac, since the full move to LLVM and the removal of GCC, it currently fakes being GCC 4.2.1)